core-bundle/contao/templates/twig/be_main.html.twig

```diff
@@ -107,7 +107,7 @@
                 {% block main_headline %}
                     <h1 id="main_headline">
                         {{ headline|raw }}
-                        {{ toggleFavorites|raw }}
+                        {{ toggleFavorites|default|raw }}
```

Comment:
Why is the controller not rendered in the template (similar to what you did with the Template Studio template)? This way it only will be generated if really output.

If this is due to the missing support in the old template engine (`be_main.html5`), we should IMHO rather add the helper function, there.

```suggestion
                        {{ render(path('contao_backend_favorites', { target_path: app.request.requestUri })) }}
```